### PR TITLE
PP: render add UID to PP if needed.

### PR DESCRIPTION
**Pull request by jlojosnegros**
When rendering manifests from PerformanceProfile we add an OwnerReference to them.
To do so we need the PerformanceProfile's UID, and if there is none we create a new one.
We need to add that new UID to PerformanceProfile to avoid problems when creating those manifests in a cluster.

IMPORTANT: Now we write ALL the input PerformanceProfiles handled to the output folder.

### CODE REVIEW

File: pkg/performanceprofile/cmd/render/render.go [hunk 1]
```diff
@@ -222,6 +223,21 @@ func render(inputDir, outputDir string) error {
 			}
 			klog.Info(fileName)
 		}
+
+		//write PerformanceProfile to output
+		ppBin, err := yaml.Marshal(pp)
+		if err != nil {
+			return fmt.Errorf("unable to marshal PerformanceProfile %s. error: %w", pp.Name, err)
+		}
+		fileName := fmt.Sprintf("%s_pp.yaml", pp.Name)
```
Comment (MarSik) [inline pkg/performanceprofile/cmd/render/render.go:232]: We should use the same name we got from the input. That will allow clean overwrite in the installer.

Comment (jlojosnegros) [reply pkg/performanceprofile/cmd/render/render.go]: yeah makes sense.
thanks for the comment

**Comment by jlojosnegros** [comment]
This approach is not good so this PR is no longer needed.
Check https://github.com/openshift/cluster-node-tuning-operator/pull/907 for the way to go.

**Pull request closed without merging**